sftplib/src/main/java/com/freiheit/fuava/sftp/SftpDownloaderJob.java

```diff
@@ -84,15 +86,26 @@ private SftpDownloaderJob() {
             final RemoteClient client,
             final RemoteConfiguration remoteConfiguration,
             final FileType fileType ) {
-        return makeDownloaderJob(
-                config, client, remoteConfiguration,
-                new SftpOldFilesMovingLatestFileFetcher(
-                    client,
-                    remoteConfiguration.getSkippedFolder(),
-                    remoteConfiguration.getProcessingFolder(),
-                    remoteConfiguration.getIncomingFolder(),
-                    fileType 
-                ) );
+        if ( remoteConfiguration.isMoveToProcessing() ) {
+            return makeDownloaderJob(
+                    config, client, remoteConfiguration,
+                    new SftpOldFilesMovingLatestFileFetcher(
+                            client,
+                            remoteConfiguration.getSkippedFolder(),
+                            remoteConfiguration.getProcessingFolder(),
+                            remoteConfiguration.getIncomingFolder(),
+                            fileType
+                    ) );
```

Comment:
I would propose to just create the fetcher based on the ìsMoveToProcessing()`. This way it's clear that this is the only difference between the two cases.